Week7/app/src/main/java/com/kulloveth/covid19virustracker/ui/news/NewsViewHolder.kt

```diff
@@ -3,17 +3,26 @@ package com.kulloveth.covid19virustracker.ui.news
 import android.view.View
 import androidx.recyclerview.widget.RecyclerView
 import com.kulloveth.covid19virustracker.R
+import com.kulloveth.covid19virustracker.data.db.NewsEntity
 import com.kulloveth.covid19virustracker.model.Article
 import com.squareup.picasso.Picasso
 import kotlinx.android.synthetic.main.headline_item.view.*
 
+
 class NewsViewHolder(val view: View) : RecyclerView.ViewHolder(view) {
 
 
-    fun bindArticle(article: Article) {
-        view.title.text = article.title
-        Picasso.get().load(article.urlToImage).error(R.drawable.ic_launcher_background)
-            .placeholder(R.drawable.covid).into(view.article_image)
-        view.description.text = article.description
+    fun bindArticle(newsEntity: NewsEntity) {
+        view.title.text = newsEntity.title
+        var path: String? = newsEntity.urlToImage
+        if (path != null) {
+            if (path.isEmpty())
+                path = null
+            Picasso.get().load(path).error(R.drawable.ic_launcher_background)
+                .placeholder(R.drawable.covid).into(view.article_image)
```

Comment:
I really like the effect that the placeholder image adds to the app, it looks very professional.